### ISSUE TITLE
Create bpmndi:BPMNLabel when setting the name of an activity

### ISSUE THREAD

**Issue by MilippFeier** [bpmn-io/bpmn-js#1498]
I am trying to include the bpmn-in-color specification (https://github.com/bpmn-miwg/bpmn-in-color/blob/master/BPMN%20in%20COLOR.pdf). Current bpmnjs version is 8.7.3. 

It turns out that the XML generated by bpmnjs is different to the bpmn-in-color reference diagram (https://github.com/bpmn-miwg/bpmn-in-color/blob/master/BPMN%20In%20Color%20Sample.bpmn).

The problem is that, in order to change (only) the text color (not the border along with it), an BPMNLabel element is required. The BPMNLabel should automatically be included when, for example, a new activity is created.

In short, Shapes (especially Tasks) should automatically include "<bpmndi:BPMNLabel>...", so that coloring only text is possible.
```

`            <bpmndi:BPMNShape id="activity_xy" bpmnElement="abc" color:background-color="#FF00FF" color:border-color="#00FF00">
                <dc:Bounds x="416" y="168" width="96" height="76"/>
                <bpmndi:BPMNLabel color:color="#3366FF" labelStyle="abc">
                    <dc:Bounds height="40" width="96" x="416" y="186"/>
                </bpmndi:BPMNLabel>`
```

**Comment by barmac** [comment]
I see your point. According to the BPMN 2.0.2 spec:

> Labels are optional for BPMNShape and BPMNEdge. When there is a label, the position of the label is specified by the bounds of the BPMNLabel of the BPMNShape or BPMNEdge. Simply put, label visibility is defined by the presence of the BPMNLabel element. The bounds of the BPMNLabel are optional and always relative to the containing BPMNPlane’s origin point (see page 376). The depiction resolution tables provided below exemplify default label positions for BPMNShape kinds (sub clause 12.3.2) and BPMNEdge kinds (sub clause 12.3.3) if no BPMNLabel bounds are provided.

If I get it correctly, we could have a `bpmndi:BPMNLabel` without bounds for shapes where we don't allow to move labels (e.g. tasks, participants).

Please feel free to create a PR which implements this feature with regard to the constrained I mentioned above.

**Comment by nikku** [comment]
We do set an empty `bpmndi:BPMNLabel` in recent bpmn-js versions (you can double-check against [our demo](https://demo.bpmn.io/s/start).

**Comment by nikku** [comment]
Closed via https://github.com/bpmn-io/bpmn-js/issues/1537.
